index.html

```diff
@@ -19,92 +20,148 @@
           </li>
         </ul>
         <a href="index.html">
-          <img class="logo" src="#" alt="логотип" />
+          <img
+            class="logo"
+            src="img/logosedona.svg"
+            width="138"
+            height="70"
+            alt="логотип"
+          />
         </a>
         <ul class="navigation-list navigation-user">
           <li class="navigation-item">
             <a class="navigation-link" href="#">
+              <img
+                class="header-icon"
+                src="img/search.svg"
+                alt="Поиск"
+                width="18"
+                height="18"
+              />
               <span class="visually-hidden"> Поиск </span>
             </a>
           </li>
           <li class="navigation-item">
             <a class="navigation-link" href="#">
+              <img
+                class="header-icon"
+                src="img/like.svg"
+                alt="Поиск"
+                width="18"
+                height="17"
+              />
               <span class="visually-hidden"> Избранное </span>
             </a>
           </li>
           <li class="navigation-item">
-            <a class="navigation-link" href="#">
-              <span class="visually-hidden"> Заказать </span>
+            <a class="navigation-link-header" href="#">
+              <span class="visually-hidden"> Хочу сюда! </span>
             </a>
           </li>
         </ul>
       </nav>
     </header>
     <main>
-      <h1 class="visually-hidden">Седона</h1>
+
       <section class="main">
```

Comment:
Кстати это можно сделать хедером мэйна
```suggestion
      <header class="main">
```